package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@duffel/api",
-  "version": "4.1.0",
+  "version": "4.2.0",
```

Comment:
Heads up, you can't pick the version yourself. We use [semantic-release](https://github.com/semantic-release/semantic-release) to publish changes. Here's an [example of the automation](https://github.com/duffelhq/duffel-api-javascript/commit/43a107acbd57593a8faedabfe60a86ca0e5503bf).